clearScreen: true
displayIntention: true
displayInferenceValues: false
displaySimAgents: false